services/cloudapi/cloudapiserver/server.go

```diff
@@ -121,6 +179,80 @@ func (ss *Service) UpdateVmState(ctx context.Context, datacenterID, serverID, ne
 	return nil
 }
 
+func (ss *Service) UpdateBootDevice(ctx context.Context, datacenterID, serverID, newBootDeviceID string) error {
```

Comment:
add a comment here to explain what happens if a cdrom is previously attached and is moved to a volume

Reply:
solved in - [90b61bb](https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/507/commits/90b61bb22669e6c0866d253d4535805c673b7c73)